packages/blocks/src/_common/components/hover/controller.ts

```diff
@@ -95,17 +95,13 @@ export class HoverController implements ReactiveController {
         }
         // Transition out
         Object.assign(this._portal.style, this._hoverOptions.transition.out);
-        this._portal.addEventListener(
-          'transitionend',
-          () => {
-            abortController.abort();
-          },
-          { signal: abortController.signal }
-        );
```

Comment:
The transition works as expected, If there is no need the leave transition in most cases, try removing the `DEFAULT_HOVER_OPTIONS.transition.out` directly.